kytos/core/kytosd.py

```diff
@@ -102,13 +105,35 @@ def main():
 
     config = KytosConfig().options['daemon']
 
+    # Configure to log uncaught exceptions to errlog file
+    sys.excepthook = exhandler
+
     if config.foreground:
         async_main(config)
     else:
         with daemon.DaemonContext():
             async_main(config)
 
 
+# pylint: disable=invalid-name
+def exhandler(exctype, value, tb):
+    """Define exception hook hanndler
+        Args:
+            exctype: exception type
+            value: value of exception
+            tb: traceback
+    """
+    #
+    # logs uncaught exceptions into the console and errlog.log
+    traceback.print_exception(exctype, value, tb)
+    # pylint: disable=logging-format-interpolation
+    logging.basicConfig(filename='errlog.log',
+                        format='%(asctime)s:%(pathname)s:'
+                        '%(levelname)s:%(message)s')
+    print(f'Uncaught Exception: {str(value)}')
+    logging.exception('Uncaught Exception: {0}'.format(str(value)))
```

Comment:
Does it really need to be printed three times: `traceback.print_exception`, `print` and `logging.exception`?

Reply:
The `print` will be removed, but the `traceback` prints it in the console and the `logging.exception` writes it to the output file.